assets/pages/new-contributions-landing/components/ButtonWithRightArrow.jsx

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+import SvgArrowRight from 'components/svgs/arrowRightStraight';
+
+type PropTypes = {
+  componentClassName: string,
+  buttonClassName: string,
+  type: string,
+  accessibilityHintId: string,
+  buttonCopy: string,
+  onClick: () => void,
+};
+// ----- Render ----- //
+
+function ButtonWithRightArrow(props: PropTypes) {
+
+  return (
+    <div className={props.componentClassName}>
```

Comment:
There may be an opportunity to generalise that component and use it elsewhere, e.g. in `ContributionSubmit`.